packages/cli/src/util/proposerConfig.ts

```diff
@@ -114,10 +113,12 @@ export function parseBuilderSelection(builderSelection?: string): routes.validat
         break;
       case "builderonly":
         break;
+      case "executionalways":
+        break;
       case "executiononly":
         break;
       default:
-        throw new YargsError("Invalid input for builder selection, check help");
+        throw Error("Invalid input for builder selection, check help");
```

Comment:
Any reason for abandoning `YargsError`? I see it being used extensively in `cli` package